axon-server-connector/src/main/java/org/axonframework/axonserver/connector/AxonServerConnectionManager.java

```diff
@@ -651,6 +657,7 @@ public String getDefaultContext() {
         private static final int DEFAULT_POOL_SIZE = 1;
 
         private AxonServerConfiguration axonServerConfiguration;
+        private Supplier<String> axonFrameworkVersionResolver = new AxonFrameworkVersionResolver();
```

Comment:
The `Builder` class and `builder()` method should define which fields are configurable and which are defaulted. As such, the addition of this `AxonFrameworkVersionResolver`/`Supplier<String>` should be added for clarity to our users.